### ISSUE TITLE
Combobox preview: ARIA 1.2 combobox does not require HTML input element

### ISSUE THREAD

**Issue by carmacleod** [IBMa/equal-access#264]
Using Preview ruleset to try out ARIA 1.2 combobox rule changes.

An ARIA 1.2 [combobox](https://w3c.github.io/aria/#combobox) can have any element as the "input" - it does not have to be an HTML input any more.
The word "input" in the spec is confusing... but it means "an ARIA [input](https://w3c.github.io/aria/#input)", which is the abstract superclass of textbox and combobox, etc.

#### Steps:
- Run [APG's Select-only combobox example](https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html).
- Run Accessibility checker with Preview rules.
- Note Violation: The text input element or number of lines is not valid for the combobox
  This shouldn't be a violation for this combobox.

The combobox is implemented with a `div` (and it's not `contenteditable` because this combobox is "select-only", i.e. the only way to set the value of the input is to select from the list, which you can do with enter, space, up, down, and by typing letters that are in the list items).

![image](https://user-images.githubusercontent.com/3331913/97567196-0e0a4700-19bd-11eb-8fd6-f567f72f7eae.png)

**Comment by tombrunet** [comment]
@carmacleod Those are the older combobox rules. You may have pulled a cached version. Easiest way to ensure it's cleared:
* Go to https://able.ibm.com/rules/archives/preview/js/ace.js
* Hit refresh and then close that tab
* Close devtools
* Go to chrome://extensions/
* Disable and re-enable the checker
* Refresh your page and then re-open devtools

**Comment by carmacleod** [comment]
Weird. I've followed your "clear cache" steps twice now, and I can't make the violation go away.
Should I still be using the "Preview Rules"? (I get the violation either way - with Latest Deployment, and with Preview Rules).

**Comment by tombrunet** [comment]
Yes - should be using preview. If I go to the latest archive (https://able.ibm.com/rules/archives/latest/js/ace.js) and search for "number of lines" I find it. If I go to preview, https://able.ibm.com/rules/archives/preview/js/ace.js that text doesn't exist, so if the preview archive isn't cached you shouldn't be able to see it. Not sure if you've restarted Chrome, but that may be what's needed?

**Comment by carmacleod** [comment]
My bad, I wasn't saving the Options after selecting Preview Rules, so I wasn't actually using the Preview Rules...

I've now saved and retested, and the ARIA 1.2 combobox rule changes are working fine for me now. Thanks for this!

Closing issue.
